### ISSUE TITLE
Test for WCAG technique H93: Ensuring that id attributes are unique on a page

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#167]
http://www.w3.org/TR/2013/NOTE-WCAG20-TECHS-20130905/H93
The objective of this technique is to avoid key errors that are known to cause problems for assistive technologies when they are trying to parse content that has the same id attribute on different elements. These errors can be avoided by making sure the Web page does not have duplicate id values. This can be done manually or by using HTML's mechanism to specify the technology and technology version, and validating the document for this condition. There are several validators that the developer can use; validation reports generally mention this type of error. The document type declaration is not strictly necessary for this type of evaluation, but specifying the document type declaration makes it easier to use a validator. 
## Test procedure
1. Check that all id attribute values are unique on the web page.
